### DISCUSSION TITLE
Using description in ModalConfig won't make the social login button in full length

### DISCUSSION THREAD

**Discussion by ssbing99** [Web3Auth/Web3Auth#580] [Help]
Im using with following settings , try to make the button became full length button based on what document said, however it still remain as icon button
```
web3AuthInstance.initModal({
          modalConfig: {
            [WALLET_ADAPTERS.OPENLOGIN]: {
              loginMethods: {
                google: {
                  name: "Google Account",
                  description: "Google Account"
                },
                twitter: {
                  name: "Twitter",
                  description: "Twitter Account"
                },
              },
            }
          }
        })
```
![image](https://user-images.githubusercontent.com/20513589/189335152-621e95c7-8598-4ede-89a1-971c151e9a86.png)

![image](https://user-images.githubusercontent.com/20513589/189335463-31c4ce2d-35c8-4ec6-8878-09329771b455.png)

My package.json
`"@web3auth/base": "^2.0.0",
    "@web3auth/web3auth": "^2.0.0",`

**Comment by rinzler99** [comment]
Hey, The feature you mentioned is applicable on torus wallet and openlogin adapter. As I can see you are trying to implement it in web3auth modal which is not possible. Thanks!

**Reply by ssbing99** [reply]
But from doc is says should be that way ? Or could you please give some advice ? Appreciate !
![image](https://user-images.githubusercontent.com/20513589/189592176-3999249d-22ad-4221-8001-788cbf8051be.png)

**Reply by rinzler99** [reply]
Yeah actually that is applicable when you try to login with external wallets or openlogin adapter, will clarify in this docs. Thanks!

**Reply by ssbing99** [reply]
So right now the "description" will not take any effects correct ? If so then I just leave it as icon button for now

**Reply by rinzler99** [reply]
Yeah right, Thanks!
